tests/test_app_views_event_rsvp.py

```diff
@@ -0,0 +1,71 @@
+"""
+This files test the event rsvp functionality
+"""
+from __future__ import absolute_import
+
+import unittest
+import json
+from app import app, db
+
+# local imports
+from config import app_config
+from .helper_methods import Helper
+
+
+class TestEventsDetails(unittest.TestCase, Helper):
+    """This class represents the Events test case"""
+
+    def setUp(self):
+        """Set up test variables."""
+        self.app = app
+        self.app.config.from_object(app_config['testing'])
+        self.client = self.app.test_client
+        self.new_event = {
+            "event": "Barbecue party",
+            "location": "nairobi",
+            "category": "Food",
+            "date": "12/12/2017"
+        }
+
+        # binds the app to the current context
+        with self.app.app_context():
+            # create all tables
+            db.create_all()
+
+    def test_rsvp_to_an_event(self):
+        """Test API can RSVP to an event (POST request)"""
+        access_token = self.get_auth_token()
+        resp = self.create_event()
+        self.assertEqual(resp.status_code, 201)
```

Comment:
You don't have to assert the test here if this is not the final assertion of the test